lib/python/qmk/info.py

```diff
@@ -686,27 +686,26 @@ def _extract_led_config(info_data, keyboard):
     cols = info_data['matrix_size']['cols']
     rows = info_data['matrix_size']['rows']
 
-    # Determine what feature owns g_led_config
     feature = None
     for feat in ['rgb_matrix', 'led_matrix']:
         if info_data.get('features', {}).get(feat, False) or feat in info_data:
             feature = feat
 
-    if feature:
-        # Only attempt search if dd led config is missing
-        if 'layout' not in info_data.get(feature, {}):
-            # Process
-            for file in find_keyboard_c(keyboard):
-                try:
-                    ret = find_led_config(file, cols, rows)
-                    if ret:
-                        info_data[feature] = info_data.get(feature, {})
-                        info_data[feature]['layout'] = ret
-                except Exception as e:
-                    _log_warning(info_data, f'led_config: {file.name}: {e}')
-
-        if info_data[feature].get('layout', None) and not info_data[feature].get('led_count', None):
-            info_data[feature]['led_count'] = len(info_data[feature]['layout'])
+            if feature:
+                # Only attempt search if dd led config is missing
+                if 'layout' not in info_data.get(feature, {}):
+                    # Process
+                    for file in find_keyboard_c(keyboard):
+                        try:
+                            ret = find_led_config(file, cols, rows)
+                            if ret:
+                                info_data[feature] = info_data.get(feature, {})
+                                info_data[feature]['layout'] = ret
+                        except Exception as e:
+                            _log_warning(info_data, f'led_config: {file.name}: {e}')
+
+                if info_data[feature].get('layout', None) and not info_data[feature].get('led_count', None):
+                    info_data[feature]['led_count'] = len(info_data[feature]['layout'])
```

Comment:
Feature will always be true here now
```suggestion
            # Only attempt search if dd led config is missing
            if 'layout' not in info_data.get(feature, {}):
                # Process
                for file in find_keyboard_c(keyboard):
                    try:
                        ret = find_led_config(file, cols, rows)
                        if ret:
                            info_data[feature] = info_data.get(feature, {})
                            info_data[feature]['layout'] = ret
                    except Exception as e:
                        _log_warning(info_data, f'led_config: {file.name}: {e}')

            if info_data[feature].get('layout', None) and not info_data[feature].get('led_count', None):
                info_data[feature]['led_count'] = len(info_data[feature]['layout'])
```